docs/cloudfoundry/cf-per-route-virtuals.rst

```diff
@@ -0,0 +1,137 @@
+- server ssl profiles.
+
+.. important:: The |cfctlr| cannot create or manage BIG-IP policies, profiles, or other configuration objects. All of the BIG-IP objects that you want to apply to the virtual server must already exist on the BIG-IP device.
+
+.. literalinclude:: /cloudfoundry/config_examples/manifest.yaml
+   :linenos:
+   :caption: Example App Manifest for cf-bigip-ctlr Service Broker
+   :lines: 1-8, 43-62
+
+.. note:: The* :code:`env.bigip` section of the Application Manifest contains the global |cfctlr| configurations. This section has been omitted from the example below for the sake of brevity.
+
+   :fonticon:`fa fa-download` :download:`Download the full example </cloudfoundry/config_examples/manifest.yaml>`
+
+
+
+Adding/Removing Service Plans
+`````````````````````````````
+
+If you want to add or remove a Service Plan for a Controller that's already registered as a Service Broker:
+
+#. Edit the Application Manifest to add or remove the desired Service Plan.
+#. `Restart the App`_ to make the new settings take effect.
+
+When you remove a Service Plan from the Application Manifest, the Controller removes all of the BIG-IP objects associated with the Plan. This is the case regardless of whether any Routes are still bound to the Service when you delete the plan.
+
+- If you remove all bound routes before you delete the Plan, the Controller will clean up the BIG-IP objects and remove the plan from the data group before the restart occurs.
+
+- If you remove the Plan without unbinding the Routes, the Controller logs will show a diff of the data group vs any incoming Plans. It will remove all BIG-IP objects associated with the deleted plan. If you try to unbind a Route *after* removing its associated Plan, the Controller takes no action on the BIG-IP.
+
+.. important:: Altering Plans that are already in effect may cause interruptions in traffic, depending on what settings have changed.
+
+Register the BIG-IP Controller as a Service Broker
+--------------------------------------------------
+
+.. attention::
+
+   The tasks in this section must be completed by a Cloud Foundry user with administrator permissions.
+
+#. `Map a Route`_ for the |cfctlr|.
+
+   .. hint:: Mapping a new Route creates a unique endpoint to use for calls to the Service Broker. This allows you to differentiate between calls to the Controller (such as Cloud Foundry health checks) and calls to the Service Broker API.
+
+   .. code-block:: console
+
+      cf map-route cf-bigip-ctlr example.com --hostname sbtest
+
+#. Register the Controller as a Service Broker.
+
+   Follow the instructions provided in `Register a Broker`_ :fonticon:`fa fa-external` as appropriate for your environment.
+   You'll need to provide the following information:
+
+   - a name for the Service Broker;
+   - a username and password for a user with permission to interact with the Service Broker API (sames as in the :code:`bigip.status` section of the Application Manifest);
```

Comment:
@f5yacobucci @dramich -- Should this be the same read-only Admin user account used by the Controller?

Reply:
This is created by the user when they create the controller, this is given to cf when you push the service broker to tell cf how to auth with the controller.

a username and password (same as in the :code:`bigip.status` section of the Application Manifest);